code/modules/mob/living/carbon/human/species/station/prommie_blob.dm

```diff
@@ -0,0 +1,466 @@
+/mob/living/simple_mob/slime/promethean
+	name = "Promethean Blob"
+	desc = "A promethean expressing their true form."
+	//ai_holder_type = null
+	color = null // Uses a special icon_state.
+	slime_color = "rainbow"
+	unity = TRUE
+	water_resist = 100 // Lets not kill the prommies
+	cores = 0
+	movement_cooldown = 3
+	//appearance_flags = RADIATION_GLOWS
+	shock_resist = 0 // Lets not be immune to zaps.
+	friendly = list("nuzzles", "glomps", "snuggles", "cuddles", "squishes") // lets be cute :3
+	melee_damage_upper = 0
+	melee_damage_lower = 0
+	player_msg = "You're a little squisher! Your cuteness level has increased tenfold."
+	heat_damage_per_tick = 20 // Hot and cold are bad, but cold is AS bad for prommies as it is for slimes.
+	cold_damage_per_tick = 20
+	//glow_range = 0
+	//glow_intensity = 0
+
+	var/mob/living/carbon/human/humanform
+	var/datum/modifier/healing
+
+	var/obj/prev_left_hand
+	var/obj/prev_right_hand
+	var/human_brute = 0
+	var/human_burn = 0
+	var/is_wide = FALSE
+	var/rad_glow = 0
+
+	var/list/default_emotes = list(
+		/decl/emote/audible/squish,
+		/decl/emote/audible/chirp,
+		/decl/emote/visible/bounce,
+		/decl/emote/visible/jiggle,
+		/decl/emote/visible/lightup,
+		/decl/emote/visible/vibrate,
+		/decl/emote/visible/flip,
+		/decl/emote/visible/spin,
+		/decl/emote/visible/floorspin
+	)
+/mob/living/simple_mob/slime/promethean/Initialize(mapload, null)
+	//verbs -= /mob/living/proc/ventcrawl
+	verbs += /mob/living/simple_mob/slime/promethean/proc/prommie_blobform
+	verbs += /mob/living/proc/set_size
+	verbs += /mob/living/proc/hide
+	verbs += /mob/living/simple_mob/proc/animal_nom
+	verbs += /mob/living/proc/shred_limb
+	verbs += /mob/living/simple_mob/slime/promethean/proc/toggle_expand
+	verbs += /mob/living/simple_mob/slime/promethean/proc/prommie_select_colour
+	verbs += /mob/living/simple_mob/slime/promethean/proc/toggle_shine
+	update_mood()
+	if(rad_glow)
+		rad_glow = CLAMP(rad_glow,0,250)
+		set_light(max(1,min(5,rad_glow/15)), max(1,min(10,rad_glow/25)), color)
+	return ..()
+
+/mob/living/simple_mob/slime/promethean/update_icon()
+	icon_living = "[icon_state_override ? "[icon_state_override] slime" : "slime"] [is_wide ? "adult" : "baby"][""]"
+	..()
+
+/mob/living/simple_mob/slime/promethean/Destroy()
+	humanform = null
+	vore_organs = null
+	vore_selected = null
+	set_light(0)
+	return ..()
+
+/mob/living/simple_mob/slime/promethean/Stat()
+	..()
+	if(humanform)
+		humanform.species.Stat(humanform)
+
+/mob/living/simple_mob/slime/promethean/handle_special() // Should disable default slime healing, we'll use nutrition based heals instead.
+	if(rad_glow)
+		rad_glow = CLAMP(rad_glow,0,250)
+		set_light(max(1,min(5,rad_glow/15)), max(1,min(10,rad_glow/25)), color)
+	else
+		set_light(0)
+	return
+
+//Constructor allows passing the human to sync damages
+/mob/living/simple_mob/slime/promethean/New(var/newloc, var/mob/living/carbon/human/H)
+	..()
+	if(H)
```

Comment:
These special promethean slimes should probably delete themselves if they don't get passed a human form.

Reply:
Sure thing, should help prevent weird edge cases.